src/settings.js

```diff
@@ -710,9 +710,11 @@ var EXPORT_EXCEPTION_HANDLING_HELPERS = false;
 
 // When this is enabled, exceptions will contain stack traces and uncaught
 // exceptions will display stack traces upon exiting. This defaults to true when
-// ASSERTIONS is enabled. This option is for users who want exceptions' stack
-// traces but do not want other overheads ASSERTIONS can incur. This currently
-// works only for Wasm exceptions (-fwasm-exceptions).
+// ASSERTIONS is enabled and when building with either -fexceptions or
+// -fwasm-exceptions. This option is for users who want exceptions' stack
```

Comment:
I think its probably enough to say "This defaults to true when ASSERTIONS is enabled." and just end that sentence there.

Reply:
4abccbc27077f67b0e0f9c3eb97866d3689ad9ec